internal/store/store.go

```diff
@@ -352,6 +370,12 @@ func (s *Store) PushBlueprint(bp blueprint.Blueprint, commitMsg string) {
 			s.BlueprintsChanges[bp.Name] = make(map[string]blueprint.Change)
 		}
 		s.BlueprintsChanges[bp.Name][commit] = change
+
```

Comment:
I think that this isn't validating new blueprint versions, just passing them through. This could result in problems for anything else expecting a valid semver version in the blueprint.